app/models/ticity_search.rb

```diff
@@ -1,3 +1,5 @@
 class TicitySearch < ApplicationRecord
     belongs_to :user
+    validates :housing, :costofliving, :startups, :capital, :travel, :commute, :business, :safety, :healthcare, :education, :environment, :economy, :taxation, :internetaccess, :leisure, :tolerance, :outdoors, presence: true
```

Comment:
Not a blocker, but if a line gets this long you should split it onto multiple lines. I think a good rule of thumb is that if you have to scroll horizontally to see it in your editor, it's too long.